### PR TITLE
add the Princeton IRB directly into the Rally add-on

### LINKED ISSUE #145: put first study IRB directly into the Core Add-On
Define the partner terms consent overlay updates, content and templatization between studies

add-on_ join a current study (study consent overlay)

### PR DESCRIPTION
**Pull request by hamilton**
closes #145.

This PR needs an engineering review. The copy _may_ change before merging. I just want to make sure the engineering part checks out okay.

UPDATE: Princeton team contact to review the flow shown [here](https://quirky-heyrovsky-1f64bb.netlify.app/iframe.html?id=core-add-on-user-flows--main-flow).

Checklist for reviewer:

- [ ] The description should reference a bug or github issue, if relevant.
- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 * [#392](https://github.com/mozilla-rally/rally-core-addon/pull/392): Implement the new consent logic to prevent running side-loaded studies.
 
+* [#383](https://github.com/mozilla-rally/rally-core-addon/pull/383): Adds IRB consent notice into the Core Add-On.
```
Comment (Dexterp37) [inline CHANGELOG.md:8]:
```suggestion
* [#383](https://github.com/mozilla-rally/rally-core-addon/pull/383): Adds IRB consent notice into the Core Add-On.
```

Let's not add blank lines between entries. I fixed this in another PR, you'll get a merge conflict once you rebase :)

File: src/routes/current-studies/StudyCard.svelte [hunk 2]
```diff
@@ -137,18 +140,12 @@ $: isActive = $activeKey !== undefined && $activeKey === key;
...
-                veniam.
-                </p>
-                <p>TBD approach based on explorations with SimplySecure.</p>
-            {/if}
+            <!-- Bake in the Princeton IRB. Once we have more studies, we will key this
+                 by the study id.
+            -->
+            <IRBWindow>
+                <svelte:component this={irb['princeton-study']} />
+            </IRBWindow>
```
Comment (Dexterp37) [inline src/routes/current-studies/StudyCard.svelte:148]: Can we do this just for the princeton addon id? This way we could still test other consent notices in our local builds.

File: stories/studies/StudyJoinFlow.svelte [hunk 3]
```diff
@@ -11,7 +11,7 @@
   // add one nicer study with all the information.
   const nicerStudy = {
     addonId: 'nicer-study-example@mozilla.org',
-    name: "Rally Core Study",
+    name: "Test STudy",
```
Comment (Dexterp37) [inline stories/studies/StudyJoinFlow.svelte:14]:
```suggestion
    name: "Test Study",
```

**Review by Dexterp37 — CHANGES_REQUESTED** [review]

File: src/routes/current-studies/StudyCard.svelte (hunk 2 above)
Comment (hamilton) [reply src/routes/current-studies/StudyCard.svelte:148]: Yes, that is the ultimate intent. Given that we don't have any other studies or consent notices, and given that we don't have the princeton addon id quite yet to do this part, I'm inclined to leave this as-is until we have more studies and more bandwidth to redesign how we're doing this part.

Do you have any suggestions on how to proceed?

Comment (Dexterp37) [reply src/routes/current-studies/StudyCard.svelte:148]: Ok

**Review by Dexterp37 — APPROVED** [review]

**Pull request merged**